apps/tx-builder/src/hooks/useServices/interfaceRepository.ts

```diff
@@ -72,9 +72,16 @@ class InterfaceRepository {
           return false;
         }
 
+        if(e?.type.toLowerCase() === 'event') {
+          return false
+        }
+
         return !e.constant;
       })
       .map((m: any) => {
+        if(m.type === 'fallback' || m.type === 'receive') {
+          return { inputs: [], name: m.type === 'fallback'  ? 'fallback' : 'receive', payable: m.payable || false };
```

Comment:
Could we simplify `m.type === 'fallback'  ? 'fallback' : 'receive'` to `m.type`, as we check before that it is either one of those two strings?

Reply:
I improved it, please check it

Reply:
looks good